sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/theme/Color.kt

```diff
@@ -2,7 +2,6 @@ package co.nimblehq.sample.compose.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-@Suppress("MagicNumber")
```

Comment:
Still need to [update](https://detekt.dev/docs/introduction/compose/) `detekt-rules.yml` tho 😉

Reply:
Thank you, on the way 😄